torch/csrc/Module.cpp

```diff
@@ -2089,6 +2095,46 @@ Call this whenever a new thread is created in order to propagate values from
       },
       "Checks if a tensor's data pointer is COW");
 
+  py_module.def(
+      "_get_cudnn_batch_norm_reserve_space_size",
+      [](const at::Tensor& input) {
+#ifdef USE_CUDA
+#if AT_CUDNN_ENABLED()
+        return at::native::_get_cudnn_batch_norm_reserve_space_size(input);
```

Comment:
This function is already properly ifdefed away in ATen, no need to do it again here

Reply:
If the cudnn file is only added when building with cudnn, you can remove the ifdef from there then